src/astx/asciiviz.py

```diff
@@ -0,0 +1,72 @@
+"""
+AST to ascii art Conversion Module.
+
+This module provides utilities for converting an Abstract Syntax Tree (AST),
+represented as a nested Python dictionary, to an ascii graph. The graph can be
+displayed directly in the terminal window.
+
+"""
+from typing import Dict, List, Tuple
```

Comment:
```suggestion
from __future__ import annotations
```

`Dict, List, Tuple` are deprecated. Now, please import annotations from `__futures__` and annotate with these words in lower case, for example:

```python
def get_nodes_edges(
    dic: dict[str, str],
    nodes: list[str] = [],
    edges: list[str] = [],
) -> tuple[list[str], list[str]]:
   ...
   ```